app/lib/swagger_docs.rb

```diff
@@ -1716,6 +1747,11 @@ def schema
                     description: "Amount of national insurance paid - normally negative, but can be positive for a refund",
                     example: -5.24,
                   },
+                  prisoner_levy: {
```

Comment:
an unfortunate duplication - is it possible to remove?

Reply:
Actually not. Income object in `employment_details.income` has an additional required attribute `receiving_only_statutory_sick_or_maternity_pay`